### PR TITLE
Adding generic parametrizable offline_node.launch

### PR DESCRIPTION
**Pull request by sebastianklose**
And adapting offline_backpack_(2d|3d).launch to use it.

This will be useful for parametrizing evaluation runs as well.

### CODE REVIEW

File: cartographer_ros/launch/offline_backpack_3d.launch [hunk 1]
```diff
@@ -15,26 +15,20 @@
...
-      required="$(arg no_rviz)">
-    <remap from="points2_1" to="horizontal_laser_3d" />
-    <remap from="points2_2" to="vertical_laser_3d" />
-  </node>
+  <remap from="points2_1" to="horizontal_laser_3d" />
+  <remap from="points2_2" to="vertical_laser_3d" />
+  <include file="$(find cartographer_ros)/launch/offline_node.launch">
+    <arg name="bag_filenames" value="$(arg bag_filenames)"/>
+    <arg name="no_rviz" value="$(arg no_rviz)"/>
+    <arg name="rviz_config" value="$(find cartographer_ros)/configuration_files/demo_3d.rviz"/>
```
Comment (kdaun) [inline cartographer_ros/launch/offline_backpack_3d.launch]: Same as 2D.

File: cartographer_ros/launch/offline_backpack_2d.launch [hunk 2]
```diff
@@ -15,25 +15,19 @@
...
-          -urdf_filenames $(find cartographer_ros)/urdf/backpack_2d.urdf
-          -bag_filenames $(arg bag_filenames)"
-      output="screen">
-    <remap from="echoes" to="horizontal_laser_2d" />
-  </node>
+  <remap from="echoes" to="horizontal_laser_2d"/>
+  <include file="$(find cartographer_ros)/launch/offline_node.launch">
+    <arg name="bag_filenames" value="$(arg bag_filenames)"/>
+    <arg name="no_rviz" value="$(arg no_rviz)"/>
+    <arg name="rviz_config" value="$(find cartographer_ros)/configuration_files/demo_2d.rviz"/>
```
Comment (kdaun) [inline cartographer_ros/launch/offline_backpack_2d.launch]: For consistency with other parameters, make `rviz_config` an argument with `"$(find cartographer_ros)/configuration_files/demo_2d.rviz"` as default value.

**Comment by sebastianklose** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user sebastianklose. Merge queue now has a length of 1.

**Pull request merged**